### ISSUE TITLE
F26 is suspicious (1.3.3 Sensory Characteristics)

### ISSUE THREAD

**Issue by kimviens** [w3c/wcag#4115]
Hi WCAG community,
[F26: Failure of Success Criterion 1.3.3 due to using a graphical symbol alone to convey information](https://www.w3.org/WAI/WCAG21/Techniques/failures/F26) has always made me feel unconfortable. Now, I think I know why.

First, at the minimum, I think the technique should also apply to 1.1.1 Non-text Content. Quoted from the technique: 
> A graphical symbol may be an image, an image of text or a pictorial or decorative character symbol (glyph) which imparts information nonverbally.

If the graphical symbol is embedded in an image element, I dont think there are any technologies that are going to treat that differently than any other image, and images are covered under 1.1.1 Non-text Content.

And then this confuses me further, and I quote:
> For example, an image with a text alternative can be used instead of the glyph.

So to remedy the problem of using a glyph, an image with an alternative text can be used. But earlier we said that a graphical symbol can be an image? So, is this implying that adding an alt text to an image of a graphical symbol would succeed this criteria? Isnt that 1.1.1 Non-text Content?

To conclude, I think this criteria is straddling the line of either being about:

- Graphical symbols used to convey information must provide an alternative (which is basically 1.1.1 Non-text Content).
- Instructions that refer to graphical symbols must make sure that they are not using a sensory characteristic alone to refer to them (so like pointing to their alt text too, for example: the red circle denoting the "overdue" status).

Can we get clarifications on this technique? Also, can we clarify the relationship between 1.3.3 Sensory Characteristics and 1.1.1 Non-text Content?

**Comment by patrickhlauke** [comment]
In short, I think F26 is absolutely NOT about 1.3.3 at all. 1.3.3 has, in the past, be quite confused, something that I tried to disambiguate here https://github.com/w3c/wcag/pull/767

Think in the same vein, this technique should just be killed off, or refocused to be about 1.1.1

**Comment by fstrr** [comment]
> Is there a way for me to propose a rewrite?

Yep! Create a branch and then start working on your update. Once you're happy with the updated content, create a PR for the backlog taskforce to review.

**Comment by patrickhlauke** [comment]
If it's ok, I'm going to close this in favour of this more wider "1.3.3 understanding, F26, F14" issue https://github.com/w3c/wcag/issues/4891
